lib/fog/ovirt/requests/compute/v4/list_template_volumes.rb

```diff
@@ -3,15 +3,22 @@ module Compute
     class Ovirt
       class V4
         class Real
+          # rubocop:disable Metrics/AbcSize
           def list_template_volumes(template_id)
             template = client.system_service.templates_service.template_service(template_id).get
-
             attachments = client.follow_link(template.disk_attachments)
-
             attachments.map do |attachment|
-              ovirt_attrs client.follow_link(attachment.disk)
+              attachment_disk = client.follow_link(attachment.disk)
+              attachment = client.follow_link(attachment)
+              bootable = attachment.bootable
+              interface = attachment.interface
+              attachment_disk.bootable = bootable if attachment_disk.bootable.nil?
+              attachment_disk.interface = interface if attachment_disk.interface.nil?
+              attachment_disk.storage_domain = attachment_disk.storage_domains[0].id
```

Comment:
This is probably not possible but I would suggest checking there is something in storage_domains before trying to take the first one.